tests/conformance/conformance.go

```diff
@@ -39,6 +39,9 @@ func execute() error {
 	} else {
 		args = append(args, "--ginkgo.focus", "\\[Conformance\\]")
 	}
+	if value, exists := os.LookupEnv("CONTAINER_TAG"); exists {
+		args = append(args, "--container-tag", value)
```

Comment:
Can you burn the default tag into the conformance binary? It is nice to have a way to override it, but I think in general we want a strict coupling between the test images and the conformance binary.

Reply:
Yep ,my intent was to enable setting the tag through `CONTAINER_TAG` because it used `latest`.

Maybe we can use [bazel stamping ](https://github.com/bazelbuild/rules_go/blob/master/go/core.rst#defines-and-stamping) in order to burn the version to the binary.
From what I understand on tests/conformance.go we need to define a variable with default value  (e.g `containerTag`)
then bazel will initialize it with the variable we defined on the correspond rule.
What you think?

Reply:
Sounds great. You can have a look at `staging/src/kubevirt.io/client-go/version` and e.g. `cmd/virtctl/BUILD.bazel` which then uses `x_defs = version_x_defs()` to get the build info at link time. We could do something similar.